src/app.html

```diff
@@ -2,7 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%sveltekit.assets%/favicon.ico" />
+
+    <link rel="icon" type="image/png" href="%sveltekit.assets%/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/svg+xml" href="%sveltekit.assets%/favicon.svg" />
+    <link rel="shortcut icon" href="%sveltekit.assets%/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="%sveltekit.assets%/apple-touch-icon.png" />
+    <meta name="apple-mobile-web-app-title" content="LDAF" />
+    <link rel="manifest" href="%sveltekit.assets%/site.webmanifest" />
```

Comment:
These were the recommendations output by [RealFaviconGenerator](https://realfavicongenerator.net/); here's the result of their checker tool when run against the preview deployment: https://realfavicongenerator.net/checker/1958b505-fc5b-4f78-9f5c-9e3ca081307c